cmd/app/main.go

```diff
@@ -40,10 +52,12 @@ func main() {
 		var err error
 		reg, err = fdbregistry.NewFoundationDBRegistry(*foundationDBClusterFilePath)
 		if err != nil {
-			log.Fatalf("error creating FoundationDB registry: %v\n", err)
+			log.Error("error creating FoundationDB registry", slog.Any("error", err))
+			return
```

Comment:
I don't think you can just return here because it will cause the program to exit with status code 0.

`log.Fatalf` exits the program with status code 1